.jenkins/pipelines/OpenEnclave/releases/publish/Jenkinsfile

```diff
@@ -77,15 +60,46 @@ EOF
                 helpers.releaseDownloadLinux(release_version, "open-enclave-hostverify", 'GitHub', 'Ubuntu', os_release)
                 sh """#!/bin/bash
                     echo "Publishing OpenEnclave ${release_version}"
-                    repoclient -c ${WORKSPACE}/config-Ubuntu${os_release}.json package add open-enclave"
-                    repoclient -c ${WORKSPACE}/config-Ubuntu${os_release}.json package add open-enclave-hostverify"
+                    mv "0.17.5/Ubuntu ${os_release}/Ubuntu_${os_release}_open-enclave_${release_version}_amd64.deb" open-enclave_${release_version}_amd64.deb
+                    mv "0.17.5/Ubuntu ${os_release}/Ubuntu_${os_release}_open-enclave-hostverify_${release_version}_amd64.deb" open-enclave-hostverify_${release_version}_amd64.deb
+                    repoclient -c ${WORKSPACE}/apt-config.json package add open-enclave_${release_version}_amd64.deb
+                    repoclient -c ${WORKSPACE}/apt-config.json package add open-enclave-hostverify_${release_version}_amd64.deb
                 """
             }
         }
         cleanWs()
     }
 }
 
+/**
+ * Publish OpenEnclave Windows packages to NuGet
+ *
+ * @param release_version  The version of the Open Enclave release to publish
+ */
+
+def windowsPublishRelease(String release_version) {
+    withCredentials([string(credentialsId: "openenclave-nuget-key", variable: "nugetkey")]) {
+        def windows_version = bat(
+            script: """
+                @echo off
+                wmic os get caption | find /v "Caption"
+                """,
+            returnStdout: true
+        ).trim()
+
+        helpers.releaseDownloadWindows(release_version, 'open-enclave', 'GitHub', windows_version)
+        helpers.releaseDownloadWindows(release_version, 'open-enclave.OEHOSTVERIFY', 'GitHub', windows_version)
+        script {
+            if (params.PUBLISH_RELEASE) {
```

Comment:
Maybe better to have it so the entire function wouldn't run anything if params.PUBLISH_RELEASE is false? Going a step further, why call this function if it is false?